components/identities/actions/changeAddressModal.tsx

```diff
@@ -30,15 +31,34 @@ const ChangeAddressModal: FunctionComponent<ChangeAddressModalProps> = ({
   const [isTxSent, setIsTxSent] = useState(false);
 
   //set_domain_to_address execute
-  const set_domain_to_address_calls = {
-    contractAddress: process.env.NEXT_PUBLIC_NAMING_CONTRACT as string,
-    entrypoint: "set_domain_to_address",
-    calldata: [...callDataEncodedDomain, hexToDecimal(targetAddress)],
+  const set_domain_to_address_call = {
+    contractAddress: process.env.NEXT_PUBLIC_STARKNETID_CONTRACT as string,
+    entrypoint: "set_user_data",
+    calldata: [
+      identity?.getId() as string,
```

Comment:
No domain since you write it on your identity

Reply:
Good idea for moving callData outside, I moved it to identityChangeCalls because it changes the identity data (the calls to the naming part are just legacy to disable it).
See: https://github.com/starknet-id/app.starknet.id/pull/580/commits/bf4be29c54e76c88d43e045c23bf43a0c195e220